### ISSUE TITLE
Set ImageOS environment variable in runner images

### ISSUE THREAD

**Issue by strideynet** [actions-runner-controller/actions-runner-controller#715]
**Is your feature request related to a problem? Please describe.**

Several off the shelf actions expect the ImageOS environment variable to be populated (e.g ruby/setup):

```
Run ruby/setup-ruby@v1
  with:
    ruby-version: 2.6
    bundler: default
    bundler-cache: false
    working-directory: .
    cache-version: 0
Error: The environment variable ImageOS must be set
```

**Describe the solution you'd like**

We should set a closest approximation of the ImageOS environment variable in the docker images (e.g `ubuntu20`)

**Describe alternatives you've considered**

You can set this as part of the Runner spec.

**Comment by callum-tait-pbx** [comment]
https://github.com/actions/virtual-environments/blob/b7f276c003aea42575b52247bdb2183e355fca2f/images/linux/scripts/installers/configure-environment.sh 

We should probably set it the same way GitHub does for their ubuntu instances i.e. whatever value the below produces:

```
tee -a /etc/environment
```

**Comment by consideRatio** [comment]
This PR is fixed by #730 according to the PR

**Comment by toast-gear** [comment]
No issues reported since release, closing as resolved.
